packages/icons/src/svg/youtube-2.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="25" height="24" viewBox="0 0 25 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Vector viewport and sizes should be in the ratio 1:1. Probably Figma added `1px` extra to the `width` due to placement in the page layout.